backend/EkoFunkcje/EkoFunkcje/Features/Attachments/GetAttachmentFunction.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using AzureFunctions.Extensions.Swashbuckle.Attribute;
+using EkoFunkcje.Models.Requests;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace EkoFunkcje.Features.Attachments
+{
+    public static class GetAttachmentFunction
+    {
+        [FunctionName("GetAttachment")]
+        public static async Task<HttpResponseMessage> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "GetAttachment")]
```

Comment:
Tutaj zastanawiam się czy powinna być taka globalna funkcja wyciągająco po absolutnym URI.
Czy jest możliwe, żeby zachowując RESTowe API do komunikacji z frontem, tj.:
Metoda HTTP: 'get'
`Route="interventions/{interventionId}/attachments/{fileId}` otrzymać odpowiedni plik bez znajomosci jego absolutnego URI? Wyciąganie załączników dla konkretnej interwencji po id wydaje mi się w tym wypadku bardziej przejrzyste.

Reply:
Oczywiście pytanie czy jest to w ogóle możliwe :)

Reply:
Też mi się wydaje ze to uri nie jest najlepsze, bo będziesz musiał pamiętać to uri po stronie frontendu. Btw. tu bedzie trzeba tez dorobic sprawdzenie czy ktos ma do tego uprawnienia

Reply:
Poprawione